core-test/src/test/scala/org/bitcoins/core/protocol/transaction/TransactionTest.scala

```diff
@@ -20,7 +21,7 @@ import scala.io.Source
  * Created by chris on 7/14/15.
  */
 class TransactionTest extends FlatSpec with MustMatchers {
-  private val logger = BitcoinSLogger.logger
+  private val logger = LoggerFactory.getLogger(this.getClass.getSimpleName)
```

Comment:
Do we have a trait for this in Bitcoin-s? Wasn't there some issue we've run into in the past using this pattern?